docs/usage/configuration.md

```diff
@@ -227,29 +227,15 @@ Note that the `show-error-end` flag is always passed into mypy, so it does not n
 ## Overriding error messages
 
 All error messages from either pylint or PythonTA checkers can be overridden.
-To override the message for a specific check, you need to know the _checker class_ that checks for the message and the _module_ in which the class is defined.
-For example, to customize `E0111` (`bad-reversed-sequence`), we require knowing that this check is made in the `BasicChecker` in the `pylint.checkers.base` module.
+Custom messages are saved in a [TOML file](https://toml.io/en/), and any section headers within the TOML file will be ignored.
```

Comment:
Move the comment about section headers being ignored into the "Notes" section below (it's not super important to the reader at first glance).